### ISSUE TITLE
Video: Demo Geographic Health Survey app

### ISSUE THREAD

**Issue by elmps2018** [opendatakit/tool-suite-X#259]
Make a video demoing the [Geographic Health Survey app](https://docs.odk-x.org/episample-intro/)! There is a good guided tour documented in these docs that you could work through on video. Do not need to show the install/code/underlying files but do show all the great features

Can do without sound or with audio narration (if with audio narration please check & fix the automatic captioning).

Context: ODK-X has started a youtube channel to share videos of how to do key tasks in ODK-X. Although these tasks are often explained in the documentation, we frequently get requests for videos and some things are easier to learn watching a video than reading documentation. We are working on getting videos in English & Spanish but also welcome contributions in other languages!

**Comment by sarahelshabrawy** [comment]
Can I work on it ?

**Comment by elmps2018** [comment]
@sarahelshabrawy thanks for helping with this documentation work! I have assigned to you! Let us know any questions.

**Comment by sarahelshabrawy** [comment]
@elmps2018 I'm sorry, I'm having some issues with my android device, would you please assign it to someone else ?

**Comment by elmps2018** [comment]
@sarahelshabrawy I thought I got a message "@elmps2018 I'm sorry, I'm having some issues with my android device, would you please assign it to someone else ?" but then don't see it here, were you able to fix and still want to be assigned or should I unassign?

**Comment by sarahelshabrawy** [comment]
I wanted to give it a try once more yes , but unfortunately I'm busy and wont be able to complete it, If you would please assign it to someone else .. sorry for the delay.

**Comment by elmps2018** [comment]
@sarahelshabrawy thanks for letting me know!

**Comment by EstherPhebe** [comment]
@elmps2018 Hello, can I work on this?

**Comment by elmps2018** [comment]
@EstherPhebe just assigned you now! Thanks for working on this!

**Comment by EstherPhebe** [comment]
@elmps2018 I hope you are well? I apologize for the delay with this issue, I've had to look for new accommodations, pack up my former home and move without much help. 

I am currently working on this and need clarification concerning the [Village Geographic Survey](https://docs.odk-x.org/episample-tour-geo-survey/) Collect Screen. I have cloned the repo and pushed it to my android device and imported the census, config and place_name csv files. When I get to the collect screen, the GPS signal spinner just stays on red and the number in the box beside the location icon stays at zero. I cannot add any records because of this and I need to show how to use the module for the video.

My phone location is turned on and I have set the GPS accuracy threshold in the settings. The first image is my device and the second image is from the documentation.

![photo_2021-05-10_15-30-48](https://user-images.githubusercontent.com/46350156/117676453-783e4880-b1a5-11eb-885e-cbd1d9dc5feb.jpg)
![vgs](https://user-images.githubusercontent.com/46350156/117676554-8e4c0900-b1a5-11eb-90b9-f0b3633f15ca.png)

**Comment by elmps2018** [comment]
Hi @EstherPhebe! I am guessing what is happening is that you have to be "at" the right location to collect the data for the household. You may be able to set very big accuracy bounds (see here: https://docs.odk-x.org/episample-tour-config/) or change the GPS coordinates in the underlying data so they work for near where you are.

**Comment by EstherPhebe** [comment]
Okay, thank you @elmps2018. I'll let you know if either of the suggestions work.

**Comment by wbrunette** [comment]
@EstherPhebe what is the status on this issue?

**Comment by EstherPhebe** [comment]
@wbrunette I was not able to resolve the issues with the Collect Module on my device, I talked to Li about it but we just never got around to figuring out what was wrong and resolving it.

**Comment by Kulex001** [comment]
@elmps2018 can i work on this please?

**Comment by elmps2018** [comment]
@Asukwo-comfort - I think @EstherPhebe was working on this but having problems. @EstherPhebe can you give us an update? Should @Asukwo-comfort give it a try (but may encounter the same problems...)

**Comment by EstherPhebe** [comment]
Hi @Asukwo-comfort, you can work on this. I had specific issues when I tried to create the video but you can try at it. The guided tour documentation is here [Geographic Health Survey](https://docs.odk-x.org/episample-intro/)

Here are a couple of demo videos on the YouTube channel, https://youtu.be/qX6VJnI6JnM and https://youtu.be/KRUlRyAQQ1c. You can get an idea of what you're supposed to do.

**Comment by josdebum** [comment]
Hello @EstherPhebe, I'm interested in working on this issue

**Comment by elmps2018** [comment]
Hi @josdebum I think @Asukwo-comfort is assigned to this from the earlier conversation. If you can find an issue that doesn't have a current assignee that would be great!

**Comment by Kulex001** [comment]
@elmps2018 I mistakenly unassigned myself. Can you reassign me this please?

**Comment by wbrunette** [comment]
@Asukwo-comfort is working on other issues so removing assigned on this issue to make it available to anyone

**Comment by Leena1666** [comment]
can I work on this issue?

**Comment by wadva474** [comment]
@wbrunette can i be assigned this task? Thanks in anticipation

**Comment by elmps2018** [comment]
@wadva474 I have assigned you, please let us know any questions while you are working away.

**Comment by wadva474** [comment]
@elmps2018 , after checking out the maleria-demo branch to install the application following the guide (https://docs.odk-x.org/episample-intro/#installing-episample) 

Below is the screenshot of running the application on my end 

<img width="1024" alt="Screenshot 2022-10-12 at 09 18 00" src="https://user-images.githubusercontent.com/34124778/195289119-d81d7e3d-85b8-4d5a-a0b6-8a733caa2d62.png">
the settings button is not responsive, and the select place name drop down is not visible. 

This is the expected Layout according to the docs 
![Screenshot 2022-10-12 at 09 19 29](https://user-images.githubusercontent.com/34124778/195289453-5818320a-48d7-43f1-ae14-1cf280cb16f3.png)

i wanted to confirm if the branch is still valid?, or i still have some settings to put in place. 
Thanks

**Comment by elmps2018** [comment]
Hi @wadva474! It looks like you are testing this out in preview xlsxconverter. If you have an android phone, please try that, since the instructions call for it ("Similarly, you can push the application to your device using the [Pushing and Pulling Files](https://docs.odk-x.org/build-app/#build-app-pushing) instructions.").

**Comment by wadva474** [comment]
Hi @elmps2018 , i have been able to install the Episample application, But having issues (https://github.com/odk-x/tool-suite-X/issues/259#issuecomment-836774936) same as @EstherPhebe , I have changed my Gps location to ethopia. and i also set a big accuracy bounds (https://github.com/odk-x/tool-suite-X/issues/259#issuecomment-837256380). 

I would like to confirm if there is a default location i should set. i haven't seen anything about that in the documentation though. 

Thanks in anticipation.

**Comment by elmps2018** [comment]
Hi @wadva474 I don't know about a default location - is it still causing problems with the location set and big bounds?

**Comment by wadva474** [comment]
yes, i have added a large bound actually, but still same issue

**Comment by elmps2018** [comment]
@wadva474 is the bound large enough that it would work from your GPS location? Are you getting the exact same screenshot as @EstherPhebe?

**Comment by wadva474** [comment]
@elmps2018 , yes i am getting the exact screenshot, i added a bound (3,000,000) still just showing the red progress. 

Probable issues i am thinking about 
- The Map API key needs to be renewed (Maps are not loading in the survey screen)

**Comment by elmps2018** [comment]
@wbrunette any ideas on this one?

**Comment by wbrunette** [comment]
The map API key is tied into the build. How are you building the tools you are viewing? Are you downloading and using the official versions from the ODK-X website?

**Comment by wadva474** [comment]
@wbrunette , the Documentation states that the malaria-demo branch of the app designer repository should be installed. which i did  to get the build

**Comment by wbrunette** [comment]
I am talking about where did you get ODK-X Tables from?

**Comment by wadva474** [comment]
i got the Table from the tables master branch @wbrunette

**Comment by wbrunette** [comment]
The issue is the Map API key is tied into our official release which is cryptographic-ally signed.

Try downloading the 3 official APKs from the ODK-X website.

**Comment by wadva474** [comment]
okay,  Thanks would try that out

**Comment by wadva474** [comment]
@wbrunette @elmps2018 , I am still having the same issue after installing the official release of Tables, services and Survey. 

attached is a video recording for more insight

[tables_recording.webm](https://user-images.githubusercontent.com/34124778/197122444-145de702-afa8-462a-b595-0cc7627713e5.webm)

**Comment by wadva474** [comment]
@elmps2018 , @wbrunette Please i need Help replicating the features of the app, still having locating issues as indicated in the video . Thanks in anticipation
